src/main/java/com/coveo/AbstractFMT.java

```diff
@@ -87,8 +90,10 @@ public void execute() throws MojoExecutionException, MojoFailureException {
       }
     }
 
+    Formatter formatter = getFormatter();
+
     for (File directoryToFormat : directoriesToFormat) {
```

Comment:
why not flatten the files from multiple root directories in order to process all of them in parallel?